src/test/java/foodwhere/logic/commands/RFindCommandTest.java

```diff
@@ -0,0 +1,83 @@
+package foodwhere.logic.commands;
+
+import static foodwhere.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import foodwhere.commons.core.Messages;
+import foodwhere.model.Model;
+import foodwhere.model.ModelManager;
+import foodwhere.model.UserPrefs;
+import foodwhere.model.review.NameContainsStallPredicate;
+import foodwhere.testutil.TypicalReviews;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code RFindCommand}.
+ */
+public class RFindCommandTest {
+    private Model model = new ModelManager(TypicalReviews.getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(TypicalReviews.getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        NameContainsStallPredicate firstPredicate =
+                new NameContainsStallPredicate(Collections.singletonList("first"));
+        NameContainsStallPredicate secondPredicate =
+                new NameContainsStallPredicate(Collections.singletonList("second"));
+
+        RFindCommand findFirstCommand = new RFindCommand(firstPredicate);
+        RFindCommand findSecondCommand = new RFindCommand(secondPredicate);
+
+        // same object -> returns true
+        assertTrue(findFirstCommand.equals(findFirstCommand));
+
+        // same values -> returns true
```

Comment:
Should keywords in different order also compare equal? What about different capitalisations?

Reply:
The find function returns stalls/ reviews according to stall index, hence keyword order does not matter. The parser is case-insensitive as well. I will add in test cases for these, thanks!